managers/views.py

```diff
@@ -1,8 +1,36 @@
+from django.db.models import Count
+from rest_framework import renderers
 from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from users.models import CustomUser
 from managers.models import Project
 from managers.serializers import ProjectSerializer
 
 
 class ProjectViewSet(viewsets.ModelViewSet):
     queryset = Project.objects.all()
     serializer_class = ProjectSerializer
+
+
+class ProjectsList(APIView):
```

Comment:
If you implement only `GET` method to retrieve a list, inherit from `rest_framework.generics.ListAPIView` (if there will be any problems after switching to this, please drop me a comment).

Reply:
Any more sophisticated API view than the basic one requires extra attributes and/or greatly limits view's flexibility.
Since we will be dropping REST anyway, I'll be switching it to Django's generic ListView. It causes far less problems anyway.

Reply:
Done